tests/test_rescue.py

```diff
@@ -10,569 +10,447 @@
-            illegal values
-        """
-        garbage = [12, -42.2, None]
-        with pytest.raises(ValueError):
-            rescue_plain_fx.unidentified_rats = garbage
-
-    @pytest.mark.parametrize("reason,reporter,marked", [
-        ("some reason", "UNIT_TEST[BOT]", True),
-        ("Totally not md", "Potato", False),
-        (None, None, True),
-        (None, None, False)
-    ])
-    def test_mark_for_deletion_setter_valid(self, rescue_sop_fx: Rescue, reason: str, reporter: str,
-                                            marked: bool):
-        rescue_sop_fx.marked_for_deletion.reporter = reporter
-        assert rescue_sop_fx.marked_for_deletion.reporter == reporter
+    rescue_plain_fx.lang_id = test_input
+    assert rescue_plain_fx.lang_id == test_input
 
-        rescue_sop_fx.marked_for_deletion.reason = reason
-        assert rescue_sop_fx.marked_for_deletion.reason == reason
 
-        rescue_sop_fx.marked_for_deletion.reporter = reporter
-        assert rescue_sop_fx.marked_for_deletion.reporter == reporter
+def test_set_unidentified_rats_garbage_in_list(rescue_plain_fx: Rescue):
+    """
+    Verifies a ValueError is raised if the list passed to Rats.unidentified_Rats contains
+    illegal values
+    """
+    garbage = [12, -42.2, None]
+    with pytest.raises(ValueError):
+        rescue_plain_fx.unidentified_rats = garbage
 
 
-    @pytest.mark.parametrize("reason,reporter,marked", [
-        ([], 42.2, -1),
-        (-2.1, {"Potato"}, None),
-        ([], 42, "md reason"),
-        (True, -42.2, uuid4())
-    ])
-    def test_mark_for_deletion_setter_bad_data(self, reason: str or None, reporter: str or None,
-                                               marked: bool, rescue_sop_fx: Rescue):
-        """
-        Verifies setting the mark for deletion property succeeds when the data is valid
+def test_set_unidentified_rats_incorrect_type(rescue_plain_fx: Rescue):
+    """
+    Verifies a TypeError is raised if another type other than list is passed.
+    """
+    with pytest.raises(TypeError):
+        rescue_plain_fx.unidentified_rats = 'Snozzberry, Wonka, Doc'
+
+
+@pytest.mark.parametrize("reason,reporter,marked", [
+    ([], 42.2, -1),
+    (-2.1, {"Potato"}, None),
+    ([], 42, "md reason"),
+    (True, -42.2, uuid4())
+])
+def test_mark_for_deletion_setter_bad_data(reason: str or None, reporter: str or None,
+                                           marked: bool, rescue_sop_fx: Rescue):
+    """
+    Verifies setting the mark for deletion property succeeds when the data is valid
 
         Args:
             rescue_sop_fx (): plain rescue fixture
             reason (str): md reason
             reporter(str) md reporter
-        """
-        with pytest.raises(TypeError):
-            rescue_sop_fx.marked_for_deletion.reason = reason
+    """
+    with pytest.raises(TypeError):
+        rescue_sop_fx.marked_for_deletion.reason = reason
 
         with pytest.raises(TypeError):
```

Comment:
inner `pytest.raises` unreachable.

Reply:
```py
def test_mark_for_deletion_setter_bad_data(reason: str or None, reporter: str or None,
                                           marked: bool, rescue_sop_fx: Rescue):
    """
    Verifies setting the mark for deletion property succeeds when the data is valid

        Args:
            rescue_sop_fx (): plain rescue fixture
            reason (str): md reason
            reporter(str) md reporter
    """
    with pytest.raises(TypeError):
        rescue_sop_fx.marked_for_deletion.reason = reason

    with pytest.raises(TypeError):
        rescue_sop_fx.marked_for_deletion.reporter = reporter

    with pytest.raises(TypeError):
        rescue_sop_fx.marked_for_deletion.marked = marked

    assert rescue_sop_fx.marked_for_deletion.marked is False
    assert rescue_sop_fx.marked_for_deletion.reason != reason
    assert rescue_sop_fx.marked_for_deletion.reporter != reporter
```

Verified these inner with statements are reachable by changing TypeError to ValueError, and verifying with failed test by wrong Exception with pytest.raised(), with the above format.  The unreachable inners seem to have been due to indent only.